rkt/api_service.go

```diff
@@ -320,93 +315,73 @@ func getBasicPod(p *pod) (*v1alpha.Pod, *schema.PodManifest, error) {
 		pod.State = v1alpha.PodState_POD_STATE_GARBAGE
 	default:
 		pod.State = v1alpha.PodState_POD_STATE_UNDEFINED
-		return pod, nil, nil
+		return pod, manifest
 	}
 
 	createdAt, err := p.getCreationTime()
 	if err != nil {
-		return nil, nil, err
+		stderr.PrintE(fmt.Sprintf("failed to get the creation time for pod %q", p.uuid), err)
+	} else if !createdAt.IsZero() {
+		pod.CreatedAt = createdAt.UnixNano()
 	}
 
 	startedAt, err := p.getStartTime()
 	if err != nil {
-		return nil, nil, err
-	}
+		stderr.PrintE(fmt.Sprintf("failed to get the start time for pod %q", p.uuid), err)
+	} else if !startedAt.IsZero() {
```

Comment:
Is testing for `IsZero()` necessary? Wouldn't `UnixNano()` return zero as the default value anyway?